db/schema.rb

```diff
@@ -87,6 +87,14 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "credits", force: :cascade do |t|
+    t.decimal "cost"
```

Comment:
i think we lose this cost and establish a constant that sets cost

Reply:
and now im even wondering if we need a credits model. could it just be an integer attribute on company?